languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34110,58 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="ENUMERATION" name="enumeration" default="temp_off">
+      <rule>
```

Comment:
Please take a closer look at the rule editor results — I just discovered another pitfall:

> L'ensemble des techniques et des matériaux provient de pays africains.

This is also `D N D N et D N`